doc/presentation/start_presentation.txt

```diff
@@ -12,7 +12,10 @@ OSGeoLive
 
     .. rv_small::
 
-        Press "s" for slide notes, ESC to zoom out, SPACE / arrow keys to navigate.
+        Press ESC to zoom out, SPACE / arrow keys to navigate, s for slide notes.
+    .. rv_small::
+
+        Note: Pop-ups are blocked in some browsers due to security settings. Try firefox or safari.
```

Comment:
Firefox and Safari are names and need capital letters.

Reply:
Thanks, good catch.